### DISCUSSION TITLE
Classic WoW Masque

### DISCUSSION THREAD

**Discussion by matm2** [SFX-WoW/Masque#203] [Issues] [answered]
For default action bars, when the addon is loaded in the game under "Texture setting" then "Global" when you choose a skin it does not change the bar skin. Please fix! The classic wow actions bars with maque plugin would be beautiful :) 
Did the same for Bartender 4.

**Comment by StormFX** [comment, marked as answer]
It works fine with Bartender 4. It does _not_ work with the default action bars. This is clearly stated on every download/project page and in the in-game options menu.

**Comment by tfadgreef** [comment]
hi m8, very nice add-on. I use Bartender 4, ElvUI classic and Masque. However, everytime I login I need to go to the masque addon again and select the skin I want. For some reason it does not remember the skin I want and resorts back to standard blizzard skins.

**Comment by StormFX** [comment]
Don't hijack issues with unrelated problems. That said, make sure the "AddonSkins" component of ElvUI is disabled.

**Comment by tfadgreef** [comment]
That did the trick thanks
